src/makeRobot.js

```diff
@@ -38,7 +38,73 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    get info() {
+      return 'name: ' + this.name + ', chip version: '
+      + this.version + ', wheels: ' + this.wheels;
```

Comment:
it could work this way
```suggestion
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
```

Reply:
ok. Thanks. But the mentor above told me to use concatenation. So...i did it

https://github.com/mate-academy/js_task_make_robot/pull/2050#discussion_r1163759014